scripts/Run-Tests.ps1

```diff
@@ -34,27 +34,15 @@ Import-Module -Force $PSScriptRoot\Projects.psm1
 
 $ScriptName = $([io.Path]::GetFileNameWithoutExtension($PSCommandPath))
 
-$ReporterOption = $null
-if ($AppVeyor) {
-    $ReporterOption = "-appveyor"
-}
-
-$TestRunnerRootPath = "$NuGetPackageRoot\xunit.runner.console\2.3.1\tools\"
-
 $failedTestProjects = @()
 foreach ($project in $Projects.Tests) {
     foreach ($framework in $Frameworks.Application) {
 
         if (-not $AppVeyor -and $framework -ne "netcoreapp2.0") { continue; }
 
         Write-Information "Running tests in ${project}: $framework..."
-        Push-Location $BinRoot\${Platform}_$Configuration\$project\$framework
-        $dll = "$project" + ".dll"
-        if ($framework -eq "netcoreapp2.0") {
-            & dotnet ${TestRunnerRootPath}netcoreapp2.0\xunit.console.dll $dll $ReporterOption
-        } else {
-            & ${TestRunnerRootPath}net452\xunit.console.exe $dll $ReporterOption
-        }
+        Push-Location $SourceRoot\$project
+        & dotnet test --no-build --framework $framework --configuration $Configuration
```

Comment:
This was required to enable netcoreapp2.1 tests because the entry point has to be netcoreapp2.1 cognizant and xunit.console only releases netcoreapp2.0 DLLs.

This update brings the script more in line with the current recommendations from both [dotnet](https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-with-dotnet-test) and [xunit](https://xunit.net/docs/getting-started/netcore/cmdline). It handles correctly detecting the xunit runner based on test projects' `PackageReference`s and correctly handles the net461 case.